packages/scripts/src/bash/ci-test.sh

```diff
@@ -133,6 +133,27 @@ if [ "$CI_TEST_MIGRATIONS_LINT" = true ]; then
   cd $START_DIRECTORY
 fi
 
+if [ "$CI_TEST_DOCS_LINT" = true ]; then
+  echo "Running test suite CI_TEST_DOCS_LINT"
+  cd ./packages/docs && yarn lint:js
+  if [ $? -ne 0 ]; then
+    echo "Aborting. exit is not 0"
+    exit 1
+  fi
+
+  cd $START_DIRECTORY
+fi
+
+if [ "$CI_TEST_DOCS_UNIT" = true ]; then
+  echo "Running test suite CI_TEST_DOCS_UNIT"
+  cd ./packages/docs && yarn build
```

Comment:
this does `yarn build`. is there a test script?

Reply:
There is no test script but i think trying too build is a good test as it is now. However we could change naming and add test script that does a build and then removes the build folder

Reply:
sounds better